.github/workflows/publish.js.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node and then publish the new bundle to NPM
+
+name: Publish CD
+
+
+on:
+  workflow_dispatch:
+
+jobs:
+  quality:
```

Comment:
Can we call it "build", instead of quality? I think it's more common to identify the initial steps in CI as "build" steps